.changeset/dirty-beans-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'eslint-plugin-lit': patch
+---
+
+More linter package setup
```

Comment:
We should do empty changesets until a package is actually released. There's no benefit in this line being adding to the changelog or bumping package version that changeset will do. We have git history for what was changed until release.
```suggestion
---
---
```

Reply:
good point :)